docs/2101UserGuide.md

```diff
@@ -298,14 +298,17 @@ Examples:
 <hr class="feature-class-separator">
 
 ### Utility commands
+CCACommander provides you with powerful utility commands to make your admin tasks even more of a breeze than it was.
 
 #### Undoing a command: `undo`
 
-Undoes the previous command that the user has entered, which has changed the data within CCACommander.
+The `undo` command is the ultimate safety net for CCACommander. You no longer have to fear accidentally deleting 
+the wrong event or enrolling the wrong member. Simply enter the `undo` command to undo your previous command, 
+and it will revert to the state it was before.
```

Comment:
do you think this line (307) sounds abit ambiguous but idk how to rephrase too so i guess no change is okay also :')

Reply:
Can consider:

Simply enter the `undo` command to revert back to its previous state.

Reply:
I will change it to `return to the previous state`